asserts/ifacedecls_test.go

```diff
@@ -56,11 +63,17 @@ func attrs(yml string) map[string]interface{} {
 	if err != nil {
 		panic(err)
 	}
+
+	// note: it's important to go through snap yaml here even though we're really interested in Attrs only,
```

Comment:
Nitpick, usually those are spelled all-upper-case NOTE as this is often highlighted by editors.